R/tm_a_regression.R

```diff
@@ -14,8 +14,8 @@
 #' Regressor variables from an incoming dataset with filtering and selecting.
 #' @param response (`data_extract_spec` or `list` of multiple `data_extract_spec`)
 #' Response variables from an incoming dataset with filtering and selecting.
-#' @param default_outlier_label (`character`, optional) The default column selected to label outliers.
-#' @param default_plot_type (`numeric`, optional) Defaults to Response vs Regressor.
+#' @param default_outlier_label (`character`) optional default column selected to label outliers.
+#' @param default_plot_type (`numeric`) optional defaults to "Response vs Regressor".
```

Comment:
I'm just guessing - should that be in backticks?
```suggestion
#' @param default_plot_type (`numeric`) optional defaults to `"Response vs Regressor"`.
```

Reply:
its numeric type  and `Response vs Regressor` is label not choices that why I did not put backtick
![image](https://github.com/insightsengineering/teal.modules.general/assets/6700955/8e499bf1-ea6b-4cb2-970a-e4c3dd368f57)

Reply:
ok